.lintrunner.toml

```diff
@@ -204,9 +204,7 @@ exclude_patterns = [
     # CUDA files are also excluded.
     '**/fb/**',
     '**/*pb.h',
-    'c10/**/cuda/*pp',
     'aten/**/cuda/*pp',
-    '**/cuda/*pp',
```

Comment:
Have we tested that these get scanned btw clang-tidy now? @malfet I think we added a PR to support lintrunner on cuda, but I don't remember the status.

Reply:
I am sure CUDA sources except 'aten/**/cuda/*pp' are scanned after this PR.